dependencies.props

```diff
@@ -119,9 +125,13 @@
     <XmlUpdateStep Include="Sni">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>SniPackageVersion</ElementName>
-      <!-- Pick any of the SNI runtime packages. All the runtime packages of sni will have the same version -->
       <PackageId>runtime.win7-x64.runtime.native.System.Data.SqlClient.sni</PackageId>
     </XmlUpdateStep>
+    <XmlUpdateStep Include="MicrosoftNETCoreApp">
```

Comment:
The itemspec here should match the `RemoteDependencyBuildInfo` itemspec, `CoreSetup`. Nothing in CoreFX itself depends on this, but it's used in build from source tooling.

Sorry for the cycles on these names. :smile: